src/terminal/parser/parser-common.vcxitems

```diff
@@ -11,6 +11,7 @@
     <ClCompile Include="..\precomp.cpp">
       <PrecompiledHeader>Create</PrecompiledHeader>
     </ClCompile>
+    <ClCompile Include="..\base64.cpp" />

```

Comment:
You'll need to add `base64.cpp` to the `sources.inc` file for the parser as well :smile: thanks!

(I was going to do it myself when I reflect these changes back into Windows, but I saw that you already updated one of the sources files!)

Reply:
I had done it, but found without that it also worked, so removed it when committing. Thanks for pointing this out, I've added it back.